src/main/java/views/LeagueSimulatorPane.java

```diff
@@ -0,0 +1,220 @@
+    final ComboBox<String> comboBox14 = new ComboBox<>(FXCollections.observableList(teamNames));
+    private VBox matchBox7 = new VBox();
+
+    private Label match8 = new Label("Match 8:");
+    final ComboBox<String> comboBox15 = new ComboBox<>(FXCollections.observableList(teamNames));
+    final ComboBox<String> comboBox16 = new ComboBox<>(FXCollections.observableList(teamNames));
+    private VBox matchBox8 = new VBox();
+
+    private HBox matches7and8 = new HBox();
+
+    public LeagueSimulatorPane() {
+        connectComponents();
+        styleComponents();
+        setCallbacks();
+    }
+
+    @Override
+    void connectComponents() {
+
+        leftContainer.getChildren().addAll(title, matchesContainer, playButton, returnButton);
+
+        matchBox1.getChildren().addAll(match1, comboBox1, comboBox2);
+        matchBox2.getChildren().addAll(match2, comboBox3, comboBox4);
+        matchBox3.getChildren().addAll(match3, comboBox5, comboBox6);
+        matchBox4.getChildren().addAll(match4, comboBox7, comboBox8);
+        matchBox5.getChildren().addAll(match5, comboBox9, comboBox10);
+        matchBox6.getChildren().addAll(match6, comboBox11, comboBox12);
+        matchBox7.getChildren().addAll(match7, comboBox13, comboBox14);
+        matchBox8.getChildren().addAll(match8, comboBox15, comboBox16);
+
+        matches1and2.getChildren().addAll(matchBox1,matchBox2);
+        matches3and4.getChildren().addAll(matchBox3,matchBox4);
+        matches5and6.getChildren().addAll(matchBox5,matchBox6);
+        matches7and8.getChildren().addAll(matchBox7,matchBox8);
+
+        matchesContainer.getChildren().addAll(matches1and2,matches3and4,matches5and6,matches7and8);
+
+        this.setLeft(leftContainer);
+        this.setCenter(centerContainer);
+    }
+
+    @Override
+    void styleComponents() {
+        leftContainer.setStyle(" -fx-padding: 20 20 20 20;\n" +
+                "        -fx-alignment: top-center;\n" +
+                "        -fx-background-color: #ccc;\n" +
+                "        -fx-spacing: 15;");
+        leftContainer.setAlignment(Pos.CENTER);
+
+        //centerContainer.setAlignment(Pos.CENTER);
+        centerContainer.setStyle("-fx-alignment: top-left;\n"+
+                " -fx-padding: 20 20 20 20;\n"+
+                "        -fx-spacing: 15;");
+
+        matchesContainer.setAlignment(Pos.CENTER);
+
+        title.setStyle("-fx-font-size: 20;");
+
+        String bigButton = "    -fx-font-size: 15;\n" +
+                "    -fx-font-family: sans-serif;\n" +
+                "    -fx-pref-width: 200;\n" +
+                "    -fx-pref-height: 40;\n" +
+                "    -fx-background-radius: 16px;";
+
+        returnButton.setStyle(bigButton);
+        playButton.setStyle(bigButton);
+    }
+
+    void createLeagueSchedule() throws Exception{
+
+        Queue<Team> schedule = new LinkedList<>();
+        String[] teamNames = {comboBox1.getSelectionModel().getSelectedItem(),
```

Comment:
This should probably be shortened to a function with a for-loop that returns `teamNames` as an array.

Reply:
This ties into the previous comment about making the amount of boxes scalable. If all of the comboBoxes are stored in a datastructure such as an ArrayList (and the other types of boxes too I'd recommend), for-looping is a lot easier!